[2023-02-22T16:20:16.012+0000] {taskinstance.py:1083} INFO - Dependencies all met for <TaskInstance: csv_from_s3_to_s3_v01.test_get_file_from_s3 manual__2023-02-22T16:20:14.857952+00:00 [queued]>
[2023-02-22T16:20:16.019+0000] {taskinstance.py:1083} INFO - Dependencies all met for <TaskInstance: csv_from_s3_to_s3_v01.test_get_file_from_s3 manual__2023-02-22T16:20:14.857952+00:00 [queued]>
[2023-02-22T16:20:16.020+0000] {taskinstance.py:1279} INFO - 
--------------------------------------------------------------------------------
[2023-02-22T16:20:16.020+0000] {taskinstance.py:1280} INFO - Starting attempt 1 of 1
[2023-02-22T16:20:16.021+0000] {taskinstance.py:1281} INFO - 
--------------------------------------------------------------------------------
[2023-02-22T16:20:16.031+0000] {taskinstance.py:1300} INFO - Executing <Task(PythonOperator): test_get_file_from_s3> on 2023-02-22 16:20:14.857952+00:00
[2023-02-22T16:20:16.048+0000] {standard_task_runner.py:55} INFO - Started process 8139 to run task
[2023-02-22T16:20:16.060+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'csv_from_s3_to_s3_v01', 'test_get_file_from_s3', 'manual__2023-02-22T16:20:14.857952+00:00', '--job-id', '210', '--raw', '--subdir', 'DAGS_FOLDER/dag_cleaning_via_s3.py', '--cfg-path', '/tmp/tmpauyf21vk']
[2023-02-22T16:20:16.066+0000] {standard_task_runner.py:83} INFO - Job 210: Subtask test_get_file_from_s3
[2023-02-22T16:20:16.184+0000] {task_command.py:388} INFO - Running <TaskInstance: csv_from_s3_to_s3_v01.test_get_file_from_s3 manual__2023-02-22T16:20:14.857952+00:00 [running]> on host 22976f44c0e9
[2023-02-22T16:20:16.265+0000] {taskinstance.py:1509} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=csv_from_s3_to_s3_v01
AIRFLOW_CTX_TASK_ID=test_get_file_from_s3
AIRFLOW_CTX_EXECUTION_DATE=2023-02-22T16:20:14.857952+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2023-02-22T16:20:14.857952+00:00
[2023-02-22T16:20:16.267+0000] {logging_mixin.py:137} INFO - file name titanic.csv
[2023-02-22T16:20:16.279+0000] {base.py:73} INFO - Using connection ID 'AWS' for task execution.
[2023-02-22T16:20:16.280+0000] {connection_wrapper.py:334} INFO - AWS Connection (conn_id='AWS', conn_type='aws') credentials retrieved from login and password.
[2023-02-22T16:20:17.246+0000] {python.py:177} INFO - Done. Returned value was: True
[2023-02-22T16:20:17.322+0000] {taskinstance.py:1323} INFO - Marking task as SUCCESS. dag_id=csv_from_s3_to_s3_v01, task_id=test_get_file_from_s3, execution_date=20230222T162014, start_date=20230222T162016, end_date=20230222T162017
[2023-02-22T16:20:17.409+0000] {local_task_job.py:208} INFO - Task exited with return code 0
[2023-02-22T16:20:17.460+0000] {taskinstance.py:2578} INFO - 0 downstream tasks scheduled from follow-on schedule check
